src/components/Filters/index.js

```diff
@@ -261,14 +261,21 @@ class Filters extends React.Component {
     this.props.setViews(views)
   }
 
+  openNewFilterModal () {
```

Comment:
Would it make sense to be names `showNewFilterModel` since we already have `showEditModal`?